APMrover2/APMrover2.cpp

```diff
@@ -330,4 +333,19 @@ void Rover::update_current_mode(void)
     control_mode->update();
 }
 
+#if FRSKY_TELEM_ENABLED == ENABLED
+void Rover::update_nav_info()
+{
+    AP_Frsky_Telem::NavInfo nav_info;
+    //
```

Comment:
These lines are weird.

Reply:
Hi Peter thanks for taking a look.
Could you please help me for I cannot see the "weirdness"!

Reply:
I think @peterbarker is talking about the blank lines with just `//` in it.